packages/components/src/internal/components/permissions/GroupsList.tsx

```diff
@@ -40,23 +38,16 @@ export const GroupsList: FC<Props> = memo(props => {
         <ul className="principal-detail-ul">
             {groups.length > 0 ? (
                 groups.map(group => {
-                    const url = createProductUrlFromParts(
-                        targetProductId,
-                        currentProductId,
-                        { expand: group.value },
-                        'admin',
-                        'groups'
-                    );
+                    const url = AppURL.create('admin', 'groups').addParams({ expand: group.value });
```

Comment:
nit: Consider adding a `ADMIN_KEY = 'admin'` to app/constants.ts to be consistent with other base routes.